openshift/template.yaml

```diff
@@ -30,17 +30,17 @@ objects:
           - name: THREE_SCALE_API_ACCESS_KEY
             valueFrom:
               secretKeyRef:
-                name: 3scale
+                name: 3scale-connect-api
                 key: three-scale-api-access-key
           - name: THREE_SCALE_API_ACCOUNT_ID
             valueFrom:
               secretKeyRef:
-                name: 3scale
+                name: 3scale-connect-api
                 key: three-scale-api-account-id
           - name: THREE_SCALE_API_HOST_URL
             valueFrom:
-              configMapKeyRef:
-                name: bayesian-config
+              secretKeyRef:
```

Comment:
note you're changing cm ref for secret ref here.

Reply:
yes @msrb if you'll check on rh-idev you can see all secrets added there under secret ref by @pburgene